pkg/quota/quota.go

```diff
@@ -14,7 +15,8 @@ const (
 var (
 	// UseQuota represents use quota or not.
 	UseQuota = true
-	gquota   = NewQuota("")
+	// Gquota represents global quota.
+	Gquota = NewQuota("")
```

Comment:
I am afraid we do not need to NewQuota twice. In the code, in the main function, we did once. And in the local variable `Gquota` we did twice. 

I think we can remove the second one.

Reply:
I'm afraid user just use the package function(StartQuotaDriver) without calling NewQuota(), Gquota is nil, it will cause panic.